pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -1412,6 +1412,10 @@ public CompletableFuture<List<String>> getOwnedTopicListForNamespaceBundle(Names
      * Check topic exists( partitioned or non-partitioned ).
      */
     public CompletableFuture<TopicExistsInfo> checkTopicExists(TopicName topic) {
```

Comment:
Mark the old method as deprecated and point to the new method checkTopicExistsAsync

Reply:
I did not change this method name.

Reply:
OK, Let me submit a PR to fix it.